iroh/src/endpoint.rs

```diff
@@ -1461,7 +1463,10 @@ mod tests {
 
         info!(time = ?test_start.elapsed(), "test setup done");
 
+        let barrier = Arc::new(tokio::sync::Barrier::new(2));
+
         // The server accepts the connections of the clients sequentially.
+        let s_b = barrier.clone();
```

Comment:
I'd add some comments as to why this is needed as otherwise the next reader will be puzzled.

Reply:
yeah, I am not sure yet I will leave it in, but it makes it a lot saner to debug these tests